### PR TITLE
Fixed: caret did not blink in empty textview, e.g. when on freshly opened popover

### PR DESCRIPTION
**Pull request by daboe01**
previously, the font needed predicting the caret position in an empty CPTextView was nil when placed on a popover that was opened for the first time.
This is fixed by this PR that introduces a fallback to the default system font.

### CODE REVIEW

**Comment by daboe01** [comment]
+#ready-to-commit
+bug
+AppKit

**Comment by daboe01** [comment]
+1

**Comment by daboe01** [comment]
-#new

File: AppKit/CPTextView/CPTextView.j [hunk 1]
```diff
@@ -1970,10 +1970,10 @@ var kDelegateRespondsTo_textShouldBeginEditing
 
         if (!numberOfGlyphs)
         {
-            var font = [_typingAttributes objectForKey:CPFontAttributeName];
+            var font = [_typingAttributes objectForKey:CPFontAttributeName] || [self font];
```
Comment (cacaodev) [inline AppKit/CPTextView/CPTextView.j]: In what circumstances _typingAttributes don't have (or lost) their CPFontAttributeName ? Is it something we should allow ?

**Comment by daboe01** [comment]
It did not lose the attribute. This is lazily initialized. 

Von meinem iPhone gesendet

**Comment by mrcarlberg** [comment]
I have tested it and it works. @cacaodev Do you have anything unresolved with this pull request or can I merge it.

**Comment by cacaodev** [comment]
Good for me

**Comment by mrcarlberg** [comment]
Merged! Thanks!

**Pull request merged**